reactioncommerce/reaction
# [Marketplace] Fix back button behaving strangely on PDP (#2660)

Resolves #2660 

**NB** This PR fixes the bug from #2660 where the browser back button behaves strangely on PDP where it was directing to the previously selected variant option instead of product grid when clicked. 

**How To Test**

1. Go to "Basic Reaction Product"
2. Click on Red and add to cart
3. Click on Green and add to cart
4. Click on Back Button in browser
5. Notice that you are being taken to the Product Grid.
